src/camera.cpp

```diff
@@ -467,7 +466,6 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime,
 	const bool swimming = (movement_XZ || player->swimming_vertical) && player->in_liquid;
 	const bool climbing = movement_Y && player->is_climbing;
 	if ((walking || swimming || climbing) &&
-			m_cache_view_bobbing &&
```

Comment:
Also better to have `m_cache_view_bobbing_amount != 0` at the beginning so it short-circuits the conditional when bobbing is disabled and doesn't then check up to all 3 of `walking || swimming || climbng`.